pom.xml

```diff
@@ -149,6 +149,12 @@
             <artifactId>event-api</artifactId>
             <version>${eventVersion}</version>
             <scope>provided</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.ibm.icu</groupId>
+                    <artifactId>icu4j</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
```

Comment:
There appears to be a bug with this nested dependency that manifests for Java 8 OpenJDK versions > 255.  I'm running  openjdk version "1.8.0_292" and things were failing badly.  Adding this exclusion got things to pass.  Bamboo appears to be locked on an earlier Java 8 version < 255 which is why it's continued to build fine there.